packages/svelte/src/internal/client/reactivity/equality.js

```diff
@@ -9,10 +9,8 @@ export function equals(value) {
  * @returns {boolean}
  */
 export function safe_not_equal(a, b) {
-	// eslint-disable-next-line eqeqeq
```

Comment:
i'm curious why this is no longer needed. was eqeqeq removed from recommended or something like that?

Reply:
I think that `eqeqeq` was already disabled before, and now the redundant disable directive was reported because of eslint/eslint#17879.

Reply:
oh, that's nice! great!